packages/priority-queue/src/request-idle-callback.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import 'requestidlecallback';
+
+/**
+ * @param timeOrDeadline - IdleDeadline object or a timestamp number.
+ */
+export type RequestIdleCallbackCallback = (
+	timeOrDeadline: IdleDeadline | number
+) => void;
```

Comment:
The jsdoc for the param is better inline

```ts
export type RequestIdleCallbackCallback = (
	/**
	 * @param timeOrDeadline - IdleDeadline object or a timestamp number.
	 */
	timeOrDeadline: IdleDeadline | number
) => void;
```

Also, if we move that type to a `types.ts` file and import here, we can preserver the git history for this file.